react/component.rst

```diff
@@ -0,0 +1,89 @@
+.. _component-label:
+
+======================
+Create React Component
+======================
+
+Generated App Code
+==================
+
+The following code is generated. It contains a class which is extended from
+a React component. The class has a :file:`render` method which contains JSX to
+render the view.
+
+::
+
+    import React, { Component } from 'react';
+    import logo from './logo.svg';
+    import './App.css';
+
+    class App extends Component {
+      render() {
+        return (
+          <div className="App">
+            <header className="App-header">
+              <img src={logo} className="App-logo" alt="logo" />
+              <h1 className="App-title">Welcome to React</h1>
+            </header>
+            <p className="App-intro">
+              To get started, edit <code>src/App.js</code> and save to reload.
+            </p>
+          </div>
+        );
+      }
+    }
+
+    export default App;
+
```

Comment:
Having some basic descriptions of used terms and technologies would help here (jsx, component). 
In the description it says the audience should be `new to React but have knowledge of JavaScript and ES6`....

Reply:
@pbauer. Interesting, you come up with many of the same things I've suggested to Rob on a separate branch. https://github.com/plone/training/tree/fredvd-react-additions . I created it last thursday or friday, but did not create a pull request, sent @robgietema an e-mail to get his opinion. Don't know yet what the best way is to proofread this material while still letting the original author cherry-pick changes so its not an all or nothing contribution and Rob can decide what to use. 

But five people doing the same kind of suggestions is also an over luxury.  how can we organise this?